### PR TITLE
replace approval auth prefix with approvalrequest - to be consistent

### PR DESCRIPTION
**Pull request by jlhughes**

### CODE REVIEW

**Review by lchanmann — APPROVED** [review]

File: src/foam/nanos/approval/AuthenticatedApprovalDAOAuthorizer.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ foam.CLASS({
       name: 'authorizeOnCreate',
       javaCode: `
         AuthService authService = (AuthService) x.get("auth");
-        if ( ! authService.check(x, "approval.create") ) {
+        if ( ! authService.check(x, "approvalrequest.create") ) {
```
Comment (nanoscott) [inline src/foam/nanos/approval/AuthenticatedApprovalDAOAuthorizer.js:26]: there are other instances of this.

**Review by nanoscott — CHANGES_REQUESTED** [review]

**Comment by nanoscott** [comment]
what about the references to this in nanopay/src/net/nanopay/approval/AuthenticatedApprovalDAO.js and nanopay/src/net/nanopay/auth/groupPermissionJunctions.jrl for basicUser .. i'm wondering why this code is even needed actually, isn't this just the default?

File: src/foam/nanos/approval/AuthenticatedApprovalDAOAuthorizer.js (hunk 1 above)
Comment (nanoscott) [reply src/foam/nanos/approval/AuthenticatedApprovalDAOAuthorizer.js:26]: I guess the other references are in the NANOPAY project, not here.

**Comment by nanoscott** [comment]
Looks like you go the groupPermissionJunctions.jrl in https://github.com/nanoPayinc/NANOPAY/pull/12057/files, but don't you need to update AuthenticatedApprovalDAO?

**Pull request closed without merging**